### DISCUSSION TITLE
Bug Report

### DISCUSSION THREAD

**Discussion by 4-S-H** [FakeFishGames/Barotrauma#14462] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The "despawnnow" command does not despawn bodies of NPCs spawned by events.

### Reproduction steps

1. Start a campaign
2. Spawn NPCs with an event
3. Kill NPCs
4. Use the console command "despawnnow"
5. Bodies of NPCs normally spawned in the outpost are despawned while bodies those spawned by the event are not

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
Yep, here's why https://github.com/FakeFishGames/Barotrauma/blob/master/Barotrauma/BarotraumaShared/SharedSource/Events/EventActions/SpawnAction.cs#L180
Otherwise key characters could disappear after some time

**Reply by 4-S-H** [reply]
So it's intended? Thanks.
